core/dev/dev.go

```diff
@@ -128,35 +129,24 @@ func (ctx *Context) Update() {
 }
 
 func (ctx *Context) Draw() {
-	if ctx.Window != nil {
-		ctx.Window.MakeContextCurrent()
-	}
 	gl.UseProgram(programs[trianglesProgID])
-	// Clear buffer
-	gl.ClearColor(1.0, 0.0, 0.0, 1.0)
-	gl.Clear(gl.COLOR_BUFFER_BIT)
+
 	// Render
 	gl.BindVertexArray(vaos[trianglesName])
 	gl.DrawArrays(gl.TRIANGLES, 0, numVertices[trianglesName])
 
 	// TODO(hurricanerix): We actually want to print the following to the dev window.
 	fmt.Printf("Shade Version: %s\n", gen.Version)
 	fmt.Printf("Built from: %s/commit/%s\n", gen.GitURL, gen.Hash)
-
-	// Swap Buffers
-	gl.Flush()
-	if ctx.Window != nil {
-		ctx.Window.SwapBuffers()
-	}
 }
 
 func createDevWindow() *glfw.Window {
 	var devWindow *glfw.Window
 	glfw.WindowHint(glfw.Resizable, glfw.True)
-	glfw.WindowHint(glfw.ContextVersionMajor, 4)
+	glfw.WindowHint(glfw.ContextVersionMajor, 2)
 	glfw.WindowHint(glfw.ContextVersionMinor, 1)
-	glfw.WindowHint(glfw.OpenGLProfile, glfw.OpenGLCoreProfile)
-	glfw.WindowHint(glfw.OpenGLForwardCompatible, glfw.True)
+	// glfw.WindowHint(glfw.OpenGLProfile, glfw.OpenGLCoreProfile)
+	// glfw.WindowHint(glfw.OpenGLForwardCompatible, glfw.True)
```

Comment:
They will be needed when we move to newer OpenGL versions. I just left them there so I don't have to look it up again.